.gitignore

```diff
@@ -9,7 +9,7 @@ dist/
 readml.egg-info/
```

Comment:
Global comment : 

Add to README.md that "Notebooks of examples using "readml" on different usecases can be found in the 'notebooks/' folder "